org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/constraints/variables/TypeVariable.xtend

```diff
@@ -100,9 +100,9 @@ class TypeVariable implements ITypeVariable {
 	// REVIEW Is it necessary to pass 'user'?
 	def reportErrors(ConfigurableDslValidator validator) {
 		errors.forEach [
-			log.debug('''Reporting error in «owner.debugInfo»: «message»''')
+			log.debug('''Reporting error in «variable.owner.debugInfo»: «message»''')
```

Comment:
¿Por qué es necesario tener la variable y no nos alcanza con el owner?

Reply:
Así estaba para las TypeSystemExceptions (`variable` es de it). No se, creo que da igual eso, no se en qué otros lugares se usa. Por qué lo preguntás?

Reply:
Pregunto porque yo venía asociando errores con una variable, y eso implica que el error está en el owner de la variable. (El owner es un EObject.) Y acá veo que la MNU tiene también un MessageSend y entonces dudo, ¿el message send es == al owner de la variable?
Si fuera igual... entonces no es necesario porque esa info ya la tengo. Si es distinto... ¿qué implicancia tiene que la variable tiene asociado un error que corresponde a otro nodo del AST? 
No estoy 100% seguro pero sospecho que puede traer problemas porque la variable "sabe" que tiene un error y eso corta determinados análisis en la variable errónea. Si el error está reportado en un nodo que no coincide con la variable me pregunto qué consecuencias tiene.

Reply:
Claro lo que yo digo es que agreguemos el error en la variable que corresponda, ¿no se puede?

Reply:
Sí, se pudo :+1: 
pero tuve que tocar otros lugares donde me explotó luego. Si podés checkear esos cambios me quedo tranquilo :P